src/components/Admin/Module/FieldRowView.vue

```diff
@@ -3,6 +3,13 @@
     <td />
     <td class="pl-3">
       {{ field.name }}
+      <b-badge
```

Comment:
Instead of using the badge, can you use text-muted and strike through the line of the table row?